libs/src/utils/estimateGas.ts

```diff
@@ -2,21 +2,36 @@
 // will go through
 const GAS_LIMIT_MULTIPLIER = 1.05
 
+interface Method {
```

Comment:
```suggestion
interface ContractMethod {
```
maybe for disambiguation purposes?

Reply:
def